### PR TITLE
Float support in jdbc connector

### PR DESCRIPTION
**Pull request by akshatnair**
Currently this is based off hive_float/4 branch. I will rebase it on master once float support for hive is merged

### CODE REVIEW

**Comment by akshatnair** [comment]
Please review the last commit "Float support in JDBC connectors " 0ac2767

File: presto-product-tests/src/main/resources/sql-tests/datasets/float_table_mysql.ddl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- type: jdbc
+CREATE TABLE %NAME%
+(
+  id_employee INT,
+  salary REAL,
+  bonus FLOAT,
+  tip FLOAT(2)
```
Comment (maciejgrzybek) [inline presto-product-tests/src/main/resources/sql-tests/datasets/float_table_mysql.ddl:7]: Would you add yet another type, e.g. FLOAT(25), to confirm it's translated to `double`?
As described in https://www.postgresql.org/docs/9.1/static/datatype-numeric.html ("_PostgreSQL accepts float(1) to float(24) as selecting the real type, while float(25) to float(53) select double precision._")

**Comment by maciejgrzybek** [comment]
LGTM % one comment

File: presto-product-tests/src/main/resources/sql-tests/datasets/float_table_mysql.ddl (hunk 1 above)
Comment (akshatnair) [inline presto-product-tests/src/main/resources/sql-tests/datasets/float_table_mysql.ddl:7]: sure

**Comment by akshatnair** [comment]
Closing this, will be worked on #274

**Pull request closed without merging**